test.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+DEFAULT_PORT=8080
+EXTRA_PORT=$1
+CONFIG_FILE_MAC="/usr/local/etc/nginx/nginx.conf"
+CONFIG_FILE_LINUX="/etc/nginx/nginx.conf"
+
+# Install Nginx on Linux
+InstallNL() {
```

Comment:
Change this to snake case. 
```suggestion
install_nginx_linux() {
```